src/components/PostsList.tsx

```diff
@@ -1,85 +1,64 @@
 import React from 'react';
+import classNames from 'classnames';
+import { Post } from '../types/Post';
 
-export const PostsList: React.FC = () => (
-  <div data-cy="PostsList">
-    <p className="title">Posts:</p>
+type Props = {
+  userPosts: Post[];
+  onOpenPost: (post: Post) => void,
+  currentPost: Post,
+};
 
-    <table className="table is-fullwidth is-striped is-hoverable is-narrow">
-      <thead>
-        <tr className="has-background-link-light">
-          <th>#</th>
-          <th>Title</th>
-          <th> </th>
-        </tr>
-      </thead>
+export const PostsList: React.FC<Props> = ({
+  userPosts,
+  onOpenPost,
+  currentPost,
+}) => {
+  return (
+    <div data-cy="PostsList">
+      <p className="title">Posts:</p>
 
-      <tbody>
-        <tr data-cy="Post">
-          <td data-cy="PostId">17</td>
+      <table className="table is-fullwidth is-striped is-hoverable is-narrow">
+        <thead>
+          <tr className="has-background-link-light">
+            <th>#</th>
+            <th>Title</th>
+            <th> </th>
+          </tr>
+        </thead>
 
-          <td data-cy="PostTitle">
-            fugit voluptas sed molestias voluptatem provident
-          </td>
+        <tbody>
+          {userPosts.map((post) => {
+            const { id, title } = post;
+            const iscurrentPost = currentPost.id !== id;
```

Comment:
```suggestion
            const isCurrentPost = currentPost.id !== id;
```
Don't forget about camelCase